src/shipit_pulse_listener/default.nix

```diff
@@ -64,10 +36,19 @@ EOF
       mkdir -p $out/bin
       ln -s ${mercurial'}/bin/hg $out/bin
     '';
+    dockerConfig = {
+      Env = [
```

Comment:
maybe it would be better to move this variables to mkPython function in ``nix/lib/default.nix``